### PR TITLE
Added documentation (GeometryInstance and VisualInstance)

### PR DESCRIPTION
**Pull request by TwistedTwigleg**
I've filled in the documentation for both GeometryInstance and VisualInstance.

### CODE REVIEW

File: doc/classes/GeometryInstance.xml
Comment (cbscribe) [inline doc/classes/GeometryInstance.xml]: Whenever possible, remove extra words that don't add content, ex: "currently".

When a member uses named constants, the standard has been to refer to them in the description, so the reader knows where to look. "See SHADOW_CASTING_SETTING_* constants for values."

File: doc/classes/GeometryInstance.xml
Comment (cbscribe) [inline doc/classes/GeometryInstance.xml]: it's -> its

File: doc/classes/GeometryInstance.xml
Comment (cbscribe) [inline doc/classes/GeometryInstance.xml]: Per doc guidelines, prefer possessive over "of the" constructions. 

The GeometryInstance's max LOD distance.

File: doc/classes/GeometryInstance.xml
Comment (akien-mga) [inline doc/classes/GeometryInstance.xml]: Please don't add empty lines, one line == one paragraph already. Use `git commit --amend` to modify that commit (see [PR workflow](http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html)).

File: doc/classes/GeometryInstance.xml
Comment (cbscribe) [inline doc/classes/GeometryInstance.xml]: See previous comment.

File: doc/classes/GeometryInstance.xml
Comment (cbscribe) [inline doc/classes/GeometryInstance.xml]: See previous comment.

File: doc/classes/GeometryInstance.xml
Comment (cbscribe) [inline doc/classes/GeometryInstance.xml]: See previous comment.

File: doc/classes/GeometryInstance.xml
Comment (cbscribe) [inline doc/classes/GeometryInstance.xml]: Don't use blank lines. Newlines start new paragraphs.

File: doc/classes/GeometryInstance.xml
Comment (cbscribe) [inline doc/classes/GeometryInstance.xml]: Remove comma after "If [code]true[/code]"
Remove "then"

File: doc/classes/VisualInstance.xml
Comment (cbscribe) [inline doc/classes/VisualInstance.xml]: Might be useful to indicate what "transformed" means here.

File: doc/classes/VisualInstance.xml
Comment (cbscribe) [inline doc/classes/VisualInstance.xml]: te -> the
handels -> handles

"Allows you to change" -> Sets

File: doc/classes/VisualInstance.xml
Comment (cbscribe) [inline doc/classes/VisualInstance.xml]: "[Camera]'s" - remove apostrophe, this isn't possessive.

**Review by cbscribe — CHANGES_REQUESTED** [review]
Great work! I added some corrections and a couple of requests for clarification.

**Comment by TwistedTwigleg** [comment]
Thanks @cbscribe  and @akien-mga!

I fixed the problems in VisualInstance and GeometeryInstance. Now they hopefully have no more spelling and grammar issues, along with being more clear. I fixed them in the second commit, is that okay? I can try to fix the first commit if needed.

I added documentation for World, Theme, StreamTexture, Sky, and ShortCut. They probably could use a second pair of eyes, since I'm not sure everything is 100% accurate to what they actually do (mainly Sky and Shortcut). I tried to assure they don't have problems like VisualInstance and GeometeryInstance, so hopefully everything is good there.

**Comment by cbscribe** [comment]
You will need to squash down to one commit before this can be merged - the Godot team likes to keep the commit log clean.

Also, as a general rule of thumb, try and do a separate PR for different, unrelated changes. It's easier to track and review if they're not all in one gigantic commit.

**Comment by TwistedTwigleg** [comment]
Should I squash this commit into one? Or make another PR with the additional documentation? (World, Theme, StreamTexture, Sky, and ShortCut)

**Review by cbscribe — APPROVED** [review]

**Comment by cbscribe** [comment]
Thanks for your effort!

**Pull request merged**
